File: angular/src/app/shared/services/system-configuration/system-setting/models/create-update-system-setting-dto.ts
import { Status } from './status';
import { EntityDto } from '@abp/ng.core';


export class CreateUpdateSystemSettingDto extends EntityDto<string>{
    key: string;
    value?: String;
    description?: string;
    status?: Status;
    effectDate?: Date;
    expireDate?: Date;
    isEncript: boolean;

    constructor(initialValues: Partial<CreateUpdateSystemSettingDto> = {}) {
        super(initialValues);
    }
}

File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/SystemSettings/Dto/CreateUpdateSystemSettingDto.cs
using doan.ProjectManagement.Enums;
using System;
using System.Collections.Generic;
using System.Text;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.SystemSettings.Dto
{
    public class CreateUpdateSystemSettingDto : EntityDto<Guid?>
    {
        public string Key { get; set; }
        public string Value { get; set; }
        public string Description { get; set; }
        public Status Status { get; set; }
        public DateTime? EffectDate { get; set; }
        public DateTime? ExpireDate { get; set; }
    }
}
